util-liquibase-installer/src/main/java/com/hpe/caf/utilliquibaseinstaller/Application.java

```diff
@@ -84,14 +82,13 @@ private void run(final String[] args) throws SQLException, LiquibaseException
             // if there's a problem in the command line,
             // you'll get this exception. this will report
             // an error message.
-            System.err.println(e.getMessage());
-            System.err.println("java Main [options...] arguments...");
+            logger.error(e.getMessage());
+            logger.error("java Main [options...] arguments...");
             // print the list of available options
             parser.printUsage(System.err);
-            System.err.println();
 
             // print option sample. This is useful some time
-            System.err.println(
+            logger.error(
```

Comment:
Leave this function pointing to `System.err` (since we don't want to diverge from `parser.printUsage`).

Reply:
added it back